src/Label/ReadModels/Relations/Repository/ReadRepositoryInterface.php

```diff
@@ -14,6 +15,11 @@ interface ReadRepositoryInterface
      */
     public function getLabelRelations(LabelName $labelName);
 
+    /**
+     * @return string[]
+     */
+    public function getLabelRelationsAsString(LabelName $labelName, RelationType $relationType): array;
```

Comment:
Maybe more important then the `AsString` suffix is adding `ForType`, so I think `getLabelRelationsForType` is better. The return type will be clear from the header block.